integration-tests/src/tests/client/features/orphan_chunk_state_witness.rs

```diff
@@ -207,8 +207,8 @@ fn setup_orphan_witness_test() -> OrphanWitnessTestEnv {
 fn test_orphan_witness_valid() {
     init_integration_logger();
 
-    if !checked_feature!("stable", StatelessValidation, PROTOCOL_VERSION) {
-        println!("Test not applicable without StatelessValidation enabled");
+    if ProtocolFeature::ChunkEndorsementV2.enabled(PROTOCOL_VERSION) {
```

Comment:
why do we need this? 🤔  it seems like those tests are still relevant with ChunkEndorsementV2

Reply:
Oh right, sorry, yes, let me remove it